_posts/en/newsletters/2023-07-26-newsletter.md

```diff
@@ -167,12 +167,42 @@ answers posted since our last update.*
 {% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
 {% assign bse = "https://bitcoin.stackexchange.com/a/" %}
 
-FIXME:bitschmidty
+- [How can I manually (on paper) calculate a Bitcoin public key from a private key?]({{bse}}118933)
+  Andrew Poelstra gives an overview of computation-by-hand verification
+  techniques, like [codex32][news239 codex32], before walking through how a
+  pubic key could be hand-derived from a private key, a process he estimates would
+  take at least 1500 hours, even with process optimizations.
+
+- [Why are there 17 native segwit versions?]({{bse}}118974)
+  Murch explains that [segwit][topic segwit] defined 17 values (0-16) for the [version bytes][bip341
+  witness program] field because of the existing availability of constant
+  opcodes OP_0...OP_16 in [Script][wiki script]. He notes that additional
+  numbers would require the use of less data-efficient `OP_PUSHDATA` opcodes.
+
+- [Does `0 OP_CSV` force the spending transaction to signal BIP125 replaceability?]({{bse}}115586)
+  Murch points to a [discussion][rbf csv discussion] confirming that since both
+  the `OP_CHECKSEQUENCEVERIFY` (CSV) [timelock][topic timelocks] and
+  Replace-by-Fee ([RBF][topic rbf]) are [enforced]({{bse}}87376) using the `nSequence` field, an
+  output with `0 OP_CSV` requires the spending transaction to signal
+  [BIP125][] replaceability.
+
+- [How do route hints affect pathfinding?]({{bse}}118755)
+  Christian Decker explains two reasons for why an LN recipient would provide a sender
```

Comment:
```suggestion
  Christian Decker explains two reasons an LN recipient would provide a sender
```
Or "... reasons why an ..." (delete "for").
But to me, "reasons" and "why" next to each other seems redundant.